$NetBSD: patch-texmf-dist_web2c_updmap.cfg,v 1.1 2020/05/10 03:56:44 markd Exp $

Start with no maps defined. Added by TEX_*MAP_FILES

--- texmf-dist/web2c/updmap.cfg.orig	2020-03-23 00:55:08.000000000 +0000
+++ texmf-dist/web2c/updmap.cfg
@@ -104,356 +104,3 @@ jaVariant -04
 
 
 #  (end of updmap-hdr.cfg)
-Map accanthis.map
-Map OrnementsADF.map
-Map ArrowsADF.map
-Map BulletsADF.map
-Map Alegreya.map
-Map AlgolRevived.map
-MixedMap allrunes.map
-Map almendra.map
-Map      euler.map
-MixedMap cm.map
-MixedMap cmextra.map
-MixedMap cyrillic.map
-MixedMap latxfont.map
-MixedMap symbols.map
-Map AnonymousPro.map
-Map uaq.map
-Map antt.map
-Map arabi.map
-MixedMap arabtex.map
-Map syriac.map
-Map archaicprw.map
-Map arev.map
-Map arimo.map
-MixedMap arss.map
-MixedMap artm.map
-Map bkaiu.map
-Map bsmiu.map
-Map gbsnu.map
-Map gkaiu.map
-Map ascii.map
-Map ascmac.map
-Map aspectratio.map
-Map augie.map
-Map auncial.map
-Map aurical.map
-Map uag.map
-Map ybv.map
-Map baskervaldx.map
-Map BaskervilleF.map
-MixedMap bbold.map
-Map belleek.map
-Map bera.map
-Map ybd.map
-Map beuron.map
-Map bguq.map
-Map bitter.map
-Map sqrcaps.map
-Map ubk.map
-Map boondox.map
-Map pbsi.map
-Map burmese.map
-Map cabin.map
-Map caladea.map
-Map calligra.map
-Map cantarell.map
-Map carlito.map
-Map cmin.map
-MixedMap cbgreek-full.map
-MixedMap ccpl.map
-Map ccicons.map
-Map clm.map
-Map chemarrow.map
-Map Chivo.map
-Map cinzel.map
-Map cjhebrew.map
-Map Clara.map
-Map ClearSans.map
-MixedMap cmtext-bsr-interpolated.map
-Map cm-lgc.map
-MixedMap cm-super-t1.map
-MixedMap cm-super-t2a.map
-MixedMap cm-super-t2b.map
-MixedMap cm-super-t2c.map
-MixedMap cm-super-ts1.map
-MixedMap cm-super-x2.map
-Map cmcyr.map
-Map cmexb.map
-MixedMap cmll.map
-Map cmsrb.map
-MixedMap cmupint.map
-Map Cochineal.map
-Map Coelacanth.map
-Map comfortaa.map
-Map ComicNeue.map
-Map ComicNeueAngular.map
-Map original-context-symbol.map
-Map CormorantGaramond.map
-Map countriesofeurope.map
-Map ucr.map
-Map crimson.map
-Map CrimsonPro.map
-Map cs-charter.map
-Map csfonts.map
-Map cuprum.map
-Map cyklop.map
-Map dad.map
-Map dante.map
-Map dejavu-type1.map
-Map dictsym.map
-Map Domitian.map
-Map dstroke.map
-Map droidsans.map
-Map droidsansmono.map
-Map droidserif.map
-Map DSSerif.map
-Map dutchcal.map
-Map EBGaramond.map
-Map EBGaramond-Maths.map
-Map yes.map
-Map epigrafica.map
-Map epiolmec.map
-Map erewhon.map
-MixedMap esint.map
-Map esrelation.map
-Map ESSTIX.map
-Map esvect.map
-Map ETbb.map
-MixedMap ethiop.map
-MixedMap eurosym.map
-Map fbb.map
-Map fdsymbol.map
-Map fetamont.map
-Map fge.map
-Map fira.map
-Map foekfont.map
-Map fonetika.map
-Map fontawesome.map
-Map fontawesome5.map
-Map nectec.map
-Map nf.map
-Map sipa.map
-Map tlwg.map
-Map forum.map
-Map fourier-utopia-expert.map
-Map fourier.map
-Map frcursive.map
-Map garuda-c90.map
-Map gentium-type1.map
-Map gfsartemisia.map
-Map gfsbaskerville.map
-Map gfsbodoni.map
-Map gfscomplutum.map
-Map gfsdidot.map
-Map gfsneohellenic.map
-Map gfsporson.map
-Map gfssolomos.map
-Map gillius.map
-Map go.map
-Map ugq.map
-Map hacm.map
-Map uhv.map
-Map Heuristica.map
-MixedMap hfbright.map
-MixedMap iby.map
-Map imfellEnglish.map
-Map zi4.map
-Map Acorn.map
-Map AnnSton.map
-Map ArtNouv.map
-Map ArtNouvc.map
-Map Carrickc.map
-Map Eichenla.map
-Map Eileen.map
-Map EileenBl.map
-Map Elzevier.map
-Map GotIn.map
-Map GoudyIn.map
-Map Kinigcap.map
-Map Konanur.map
-Map Kramer.map
-Map MorrisIn.map
-Map Nouveaud.map
-Map Romantik.map
-Map Rothdn.map
-Map RoyalIn.map
-Map Sanremo.map
-Map Starburst.map
-Map Typocaps.map
-Map Zallman.map
-Map InriaSans.map
-Map InriaSerif.map
-Map ipaex-type1.map
-Map iwona.map
-KanjiMap otf-@jaEmbed@.map
-KanjiMap otf-ko-@koEmbed@.map
-KanjiMap otf-sc-@scEmbed@.map
-KanjiMap otf-tc-@tcEmbed@.map
-KanjiMap otf-up-@jaEmbed@.map
-Map Junicode.map
-Map kerkis.map
-Map knitfont.map
-Map kpfonts.map
-Map kurier.map
-Map lato.map
-Map libertine.map
-Map libertinegc.map
-Map libertinus.map
-Map libertinust1math.map
-Map LibreBaskerville.map
-Map LibreBodoni.map
-Map LibreCaslon.map
-Map LibreFranklin.map
-Map yly.map
-Map linearA.map
-Map LinguisticsPro.map
-Map l7x-urwvn.map
-Map lm.map
-Map LobsterTwo.map
-MixedMap lxfonts.map
-MixedMap manfnt.map
-Map marcellus.map
-Map marvosym.map
-Map mathabx.map
-Map mdbch.map
-Map mdgreek.map
-Map mdici.map
-Map mdpgd.map
-Map mdpus.map
-Map mdput.map
-Map mdugm.map
-Map mdsymbol.map
-Map merriweather.map
-Map troff-updmap.map
-MixedMap mflogo.map
-Map miama.map
-Map mintspirit.map
-Map MnSymbol.map
-MixedMap mongolian.map
-Map Montserrat.map
-KanjiMap morisawa5.map
-MixedMap musix.map
-Map mxedruli.map
-Map nanumfonts.map
-Map unc.map
-Map newpx.map
-Map newtx.map
-Map newtxsf.map
-Map newtxtt.map
-Map niceframe.map
-Map nimbus15.map
-Map norasi-c90.map
-Map noto.map
-Map cherokee.map
-Map ocrb.map
-Map oinuit.map
-Map oasy.map
-Map OldStandard.map
-Map omega.map
-Map opensans.map
-Map overlock.map
-Map upl.map
-Map paratype-type1.map
-Map dummy-space.map
-Map phaistos.map
-MixedMap pigpen.map
-MixedMap plother.map
-MixedMap pltext.map
-Map PlayfairDisplay.map
-Map plex.map
-Map ap.map
-Map prodint.map
-Map charter.map
-Map fpls.map
-Map pazo.map
-Map utopia.map
-KanjiMap ptex-@jaEmbed@@jaVariant@.map
-Map pxfonts.map
-Map pxtx.map
-Map quattrocento.map
-Map Raleway.map
-Map recycle.map
-Map roboto.map
-Map yrd.map
-Map Rosario.map
-MixedMap rsfs.map
-Map rsfso.map
-Map skt.map
-Map sansmathaccent.map
-Map sansmathfonts.map
-Map scanpages.map
-Map ScholaX.map
-MixedMap semaf.map
-Map SkakNew.map
-Map SourceCodePro.map
-Map SourceSansPro.map
-Map SourceSerifPro.map
-Map starfont.map
-Map icelandic.map
-Map STEP.map
-Map SticksTooText.map
-Map stix.map
-Map stix2.map
-MixedMap stmaryrd.map
-Map superiors.map
-Map svrsymbols.map
-Map usy.map
-Map tabvar.map
-Map tempora.map
-Map qag.map
-Map qbk.map
-Map qcr.map
-Map qcs.map
-Map qhv.map
-Map qpl.map
-Map qtm.map
-Map qzc.map
-Map mathpple.map
-Map tfrupee.map
-Map TheanoDidot.map
-Map TheanoModern.map
-Map TheanoOldStyle.map
-Map utm.map
-Map tinos.map
-MixedMap tipa.map
-MixedMap trajan.map
-Map txfonts.map
-Map gptimes.map
-Map TXUprCal.map
-Map umj.map
-Map universalis.map
-KanjiMap uptex-@jaEmbed@@jaVariant@.map
-KanjiMap uptex-ko-@koEmbed@.map
-KanjiMap uptex-sc-@scEmbed@.map
-KanjiMap uptex-tc-@tcEmbed@.map
-MixedMap dvng.map
-Map yv1.map
-Map yv2.map
-Map yv3.map
-Map yvo.map
-Map yvt.map
-Map arevvn.map
-Map chartervn.map
-Map cmbrightvn.map
-Map concretevn.map
-Map grotesqvn.map
-Map txttvn.map
-Map urwvn.map
-MixedMap vnrother.map
-MixedMap vnrtext.map
-Map vntopia.map
-Map dgj.map
-Map dmj.map
-Map mc2j.map
-Map mcj.map
-Map mr2j.map
-Map mrj.map
-MixedMap wasy.map
-Map XCharter.map
-MixedMap xypic.map
-Map yfrak.map
-MixedMap yhmath.map
-Map uzc.map
-Map uzd.map
